### ISSUE TITLE
Git icdiff v1 v2, file name is hard to recognize 

### ISSUE THREAD

**Issue by LucianLiu6** [jeffkaufman/icdiff#134]
#### Just like the screenshot below:
![c16b555d-000d-484e-8029-7519c0da7e89](https://user-images.githubusercontent.com/16591903/39909521-d13844f2-5525-11e8-859d-fe10bf10f067.png)  
the left file name is:  
`/var/folders/_8/3z_y4bz11yd1qfth6r51_mk00000gn/T//uuUv58_cinemamovies.js`  
and the right one is:  
`/var/folders/_8/3z_y4bz11yd1qfth6r51_mk00000gn/T//awWR98_cinemamovies.js`  
#### Is there any solution to make the file name more friendly?

**Comment by LunaticoCR** [comment]
+1
@jeffkaufman

**Comment by jeffkaufman** [comment]
Is this fixed on master with ae2906d3fcd545e1 ?  If so I can put out a new release

**Comment by LucianLiu6** [comment]
Not yet

I tried the follow steps to upgrade the icdiff to e2906d3 which on the master branch right now, but also get the unrecognizable file path.
1. Uninstall the old icdiff whose version is `v1.9.1`
2. Install the icdiff whose commit-id is e2906d3 by this command
```shell
cd ${icdiff_repo} && git checkout e2906d3 && python3 setup.py install
```
3. Using the new version of icdiff to view the diff between `HEAD~1` and `HEAD`
<img width="1273" alt="icdiff-error" src="https://user-images.githubusercontent.com/16591903/61262069-817f4980-a7b6-11e9-82b0-f8521fc61694.png">


Finally, I get the unrecognizable file path again.

**Comment by LucianLiu6** [comment]
I have fixed the problem in #159 

You can have a look @LunaticoCR

**Comment by LucianLiu6** [comment]
Fixed in PR #159
